server/routes/api.js

```diff
@@ -0,0 +1,103 @@
+const express = require('express');
+const router = express.Router();
+const axios = require('axios');
+const moment = require('moment');
+const City = require('../models/City.js');
+const API_KEY = "b7ccccddb421f13c7a2a6da8f86761b4";
```

Comment:
try reading about `.env` files, usually, we put API keys in those files and not in the JS files